controllers/contacts.js

```diff
@@ -0,0 +1,108 @@
+const Joi = require("joi");
+const { HttpError, ctrlWrapper } = require("../helpers");
+const { Schema, model } = require("mongoose");
+
+const contactsSchema = new Schema({
```

Comment:
Move Model Schema into models => contacts

Reply:
Also, in this pull request or the next one, create a "contacts" folder in the "controllers" directory, where all the controllers handling "/contact" requests will reside. Divide all the controllers into files: addContact.js, updateContact.js, etc. Add a default index.js file for re-exporting the controllers.

Reply:
i think its done, check plaese